client-local/src/main/java/com/cloudera/livy/client/local/LocalClient.java

```diff
@@ -284,73 +288,66 @@ public void run() {
       // SparkSubmit will take care of that for us.
       String master = conf.get("spark.master");
       Preconditions.checkArgument(master != null, "spark.master is not defined.");
-
-      List<String> argv = Lists.newArrayList();
-
+      //---Modifying argv to be a map
+       HashMap<String,String> argv=new HashMap<String,String>();
```

Comment:
I added the HashMap because it was easier to add all parameters in one place along with the launcher creation. Modified  this with the latest PR